thelper/draw.py

```diff
@@ -703,9 +718,9 @@ def draw_bbox(image, tl, br, text, color, box_thickness=2, font_thickness=1,
     return win_name, image
 
 
-def draw_bboxes(images,                 # type: thelper.typedefs.ImageArray
-                preds=None,             # type: Optional[thelper.typedefs.BoundingBoxArray]
-                bboxes=None,            # type: Optional[thelper.typedefs.BoundingBoxArray]
+def draw_bboxes(images,
+                preds=None,
+                bboxes=None,
                 color_map=None,         # type: Optional[thelper.typedefs.ClassColorMap]
```

Comment:
Why remove the typings?

Reply:
no clue, types were bad maybe, added back in with correct ones